src/snowflake/cli/plugins/nativeapp/manager.py

```diff
@@ -100,6 +106,20 @@ def ensure_correct_owner(row: dict, role: str, obj_name: str) -> None:
         raise UnexpectedOwnerError(obj_name, role, actual_owner)
 
 
+def _get_paths_to_sync(paths_to_sync: List[Path], deploy_root: Path) -> List[str]:
```

Comment:
Yes, not visible to the customer using the CLI. But visible to the public in general since this is open source. 
Regardless, good idea to still have some information on what this function does.

Reply:
yep, internal